credit-conveyor/Application/src/main/resources/db.changelog/creditconveyor-database.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In ms Application there isn't any persistence, so liquibase scripts should be moved into Deal service. Application is stateless, Deal is statefull.